addon/components/frost-list.js

```diff
@@ -114,6 +136,36 @@ export default Component.extend({
     })
   },
 
+  @readOnly
+  @computed('componentKeyNamesForTypes')
+  isAnyTypedItemExpansion (componentKeyNamesForTypes) {
+    const componentKeyNames = this.get('componentKeyNames')
+    const itemExpansionKeyName = get(componentKeyNames, 'itemExpansion')
+
+    for (var itemType in componentKeyNamesForTypes) {
```

Comment:
I think we also want to take in consideration `itemExpansion` so this way `isExpandAllVisible` will depend only on this property ^

Reply:
I have changed `isAnyTypedItemExpansion` to `isAnyItemExpansion` and now we do not need `isExpandAllVisible` anymore because it covers the same logic.

Reply:
Now that this computed property has been changed to account for both typed item expansion and default item expansion, we do not need `isExpandAllVisible` anymore.